stackslib/src/monitoring/mod.rs

```diff
@@ -38,9 +37,8 @@ use crate::util_lib::db::{sqlite_open, tx_busy_handler, DBConn, Error as Databas
 mod prometheus;
 
 #[cfg(feature = "monitoring_prom")]
-lazy_static! {
-    static ref GLOBAL_BURNCHAIN_SIGNER: Mutex<Option<BurnchainSigner>> = Mutex::new(None);
-}
+static GLOBAL_BURNCHAIN_SIGNER: LazyLock<Mutex<Option<BurnchainSigner>>> =
```

Comment:
Rust has allowed static mutexes for a couple years now, it's not necessary to put it in a `LazyLock`